lightkurve/lightcurve.py

```diff
@@ -530,6 +530,22 @@ def __init__(self, path, quality_bitmask=KeplerQualityFlags.DEFAULT_BITMASK,
         self.quality_bitmask = quality_bitmask
         self.quality_mask = self._quality_mask(quality_bitmask)
 
+    @property
+    def hdu(self):
+        return self._hdu
+
+    @hdu.setter
+    def hdu(self, value, keys=['SAP_FLUX','SAP_QUALITY']):
+        '''Raises a ValueError exception if self.hdu does not appear to be a Target Pixel File.
```

Comment:
Change `does not appear to be a Target Pixel File.` into `does not appear to be a Kepler light curve file.`